NASA-AMMOS/aerie-ui
# Add resize handle to Model ID column header

resolves #1077

To test:
1. Load plans page
2. Verify that the `Model ID` column is now resizable
